content/en/docs/reference/using-api/_index.md

```diff
@@ -50,27 +50,30 @@ Here's a summary of each level:
 
 - Alpha:
   - The version names contain `alpha` (for example, `v1alpha1`).
-  - The software may contain bugs. Enabling a feature may expose bugs. A
-    feature may be disabled by default.
-  - The support for a feature may be dropped at any time without notice.
+  - Alpha API versions are disabled by default.
+  - The software may contain bugs. Enabling a feature may expose bugs.
+  - Support for an alpha API may be dropped at any time without notice.
   - The API may change in incompatible ways in a later software release without notice.
   - The software is recommended for use only in short-lived testing clusters,
     due to increased risk of bugs and lack of long-term support.
 
 - Beta:
   - The version names contain `beta` (for example, `v2beta3`).
+  - Built-in beta API versions are typically disabled by default (starting with new Beta APIs added since Kubernetes 1.21).
```

Comment:
Is this right?
```suggestion
  - For built-in APIs, beta versions are disabled by default, **except** for beta versions of APIs that were enabled
    before Kubernetes 1.21 (which introduced the disabled-by-default requirement)
```

Reply:
rephrased to something closer to this suggestion